src/widgets/RealtimeEffectPanel.h

```diff
@@ -0,0 +1,48 @@
+/*!********************************************************************
```

Comment:
This pair of files is still in `src/widgets` and I would rather just put it under `src` pending other folder reorganizations later.